assets/js/pages/HostsList/HostsList.test.jsx

```diff
@@ -121,6 +125,37 @@ describe('HostsLists component', () => {
         ).toBeVisible()
       );
     });
+
+    it('should show only unique SIDs', async () => {
+      const host = hostFactory.build();
+      const duplicateSID = faker.string.alpha({ casing: 'upper', count: 3 });
+      const id = faker.string.uuid();
+      const sapInstances = sapSystemApplicationInstanceFactory.buildList(2, {
+        sap_system_id: id,
+        sid: duplicateSID,
+        host_id: host.id,
+      });
+      const databaseInstances = databaseInstanceFactory.buildList(2, {
+        sid: duplicateSID,
```

Comment:
This should have the same `id`, right?
As we are doing the filtering by id:
`database_id: id`